muzzle/src/main/java/io/opentelemetry/javaagent/tooling/StaticInstrumenterHook.java

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.tooling;
+
+import java.util.Map;
+
+/**
+ * This hook can be registered in an agent extension by setting the
+ * HelperInjector.staticInstrumenterHook field. It is used to process additional classes created by
+ * the agent.
+ */
+public interface StaticInstrumenterHook {
+
+  void injectClasses(Map<String, byte[]> classnameToBytes);
+}
```

Comment:
I think this would be a bit clearer if the class was named `*Listener`, something like:

```suggestion
public interface HelperInjectorListener {

  void onInjection(Map<String, byte[]> classnameToBytes);
}
```

Reply:
You're right, thanks!